pkg/agent/flexvolume/server.go

```diff
@@ -40,13 +42,16 @@ const (
 	FlexvolumeDriver         = "rook"
 	flexvolumeDriverFileName = "rookflex"
 	usrBinDir                = "/usr/local/bin/"
```

Comment:
pkg/flexvolume/controller.go has a similar constant. https://github.com/rook/rook/blob/master/pkg/agent/flexvolume/controller.go#L52
Can you perhaps put these constants somewhere where it could be referenced by both files?

Reply:
i have already moved that const to a single location in server.go.  let me know if i'm missing something still.